docs/flow/monitoring/debugging.md

```diff
@@ -6,3 +6,56 @@ weight: 300
 ---
 
 # Debugging
+
+There are two primary ways to debug issues with Grafana Agent Flow:
+
+1. Through the Grafana Agent Flow UI
```

Comment:
```suggestion
* Through the Grafana Agent Flow UI
```

Reply:
Suggest reserving numbered lists for steps that need a specific order, otherwise use bullets.

Reply:
Right, the intention here is that people check the UI first before trying to investigate logs.

Reply:
Should that be more clear in the preceding sentence?